0_From_Code_First_To_First_Principles/1_code_first/assignment/src/interfaces/index.ts

```diff
@@ -0,0 +1,5 @@
+import type { Pool } from 'pg';
+
+export type Context = {
```

Comment:
I like this idea - passing a well-defined `Context` around to the route handlers.